include/zenoh-pico/collections/string.h

```diff
@@ -70,9 +70,9 @@ _z_string_t _z_string_null(void);
 _Bool _z_string_check(const _z_string_t *value);
 _z_string_t _z_string_make(const char *value);
 _z_string_t _z_string_n_make(const char *value, size_t len);
-_z_string_t _z_string_from_str(const char *value);
```

Comment:
it would probably be better to have names consistent with corresponding slice functions, i.e. if you change _from to _alias, it should likely be also done for slice functions. Another option is to replace _z_string_make with _z_string_from_str_copy and _z_string_n_make with _z_string_from_substr_copy.